diff --git a/node_modules/react-native-image-crop-picker/RNImageCropPicker.podspec b/node_modules/react-native-image-crop-picker/RNImageCropPicker.podspec
index 97ef08f..a469bad 100644
--- a/node_modules/react-native-image-crop-picker/RNImageCropPicker.podspec
+++ b/node_modules/react-native-image-crop-picker/RNImageCropPicker.podspec
@@ -13,7 +13,7 @@ Pod::Spec.new do |s|
   s.platform     = :ios, "8.0"
   s.dependency 'React-Core'
   s.dependency 'React-RCTImage'
-  s.dependency 'TOCropViewController', '~> 2.7.4'
+  s.dependency 'TOCropViewController', '~> 2.5.3'
   s.resource_bundles = {
     'RNImageCropPickerPrivacyInfo' => ['ios/PrivacyInfo.xcprivacy'],
   }